dashboard/lib/pd/foorm/rollup_helper.rb

```diff
@@ -45,12 +46,10 @@ def self.get_rollup_question_data(parsed_forms, question_details)
       question_details.each do |form_type, questions_per_type|
         questions[form_type] = {}
         questions_per_type.each do |question, question_data|
-          # for now we will skip questions that don't have the same type and choices/rows/columns
-          # across all versions/forms.
-          next unless validate_question(question, question_data, parsed_forms, form_type)
-          first_form_key = question_data[:form_keys].first
-          parsed_form = parsed_forms[form_type][first_form_key]
-          parsed_question_data = parsed_form[question]
+          # for now we will skip questions that don't have the same type and choices/columns
+          # across all versions/forms. We will merge rows for matrix questions if type and columns are the same
```

Comment:
super nit: would prefer capitalisation and period for all sentences to make comments a bit more readable.  When all lower case it can look like a series of bullet points rather than one sentence. :)